deploy/resources/v0.9.2/addons/clustertasks/buildah/buildah-task.yaml

```diff
@@ -0,0 +1,56 @@
+# auto generated by script/update-tasks.sh
+# DO NOT EDIT: use the script instead
+# source: https://raw.githubusercontent.com/openshift/tektoncd-catalog/release-v0.8/buildah/buildah.yaml
```

Comment:
It seems the files are downloaded from the old release branch instead of  0.9; see https://github.com/openshift/tektoncd-catalog/tree/release-v0.9

Reply:
i will update the clustertasks in a separate pr